sdk/python/feast/transformation/substrait_transformation.py

```diff
@@ -1,10 +1,9 @@
 from types import FunctionType
-from typing import Any, Dict, List
+from typing import Any
 
 import dill
 import pandas as pd
 import pyarrow
-import pyarrow.substrait as substrait  # type: ignore # noqa
```

Comment:
@shuchu I know it looks like it's not used anywhere, but this is a required import :)

Reply:
just learned from the unit-testing failures at the same time....   have added it back.